samples/boards/stm32/gui_hci_uart/prj.conf

```diff
@@ -0,0 +1,15 @@
+CONFIG_CONSOLE=n
+CONFIG_STDOUT_CONSOLE=n
+CONFIG_UART_CONSOLE=n
+CONFIG_GPIO=y
```

Comment:
Not required, as part of minimum default board configuration.

Reply:
Done.